OpenSim/ApplicationPlugins/RegionModulesController/RegionModulesControllerPlugin.cs

```diff
@@ -245,8 +245,8 @@ protected bool CheckModuleEnabled(TypeExtensionNode node, IConfig modulesConfig)
                     className = moduleParts[1];
 
                 // Match the class name if given
-                if (className != String.Empty &&
-                        node.Type.ToString() != className)
+                if (!(String.IsNullOrEmpty(className) ||
+                    node.Type.ToString() == className))
```

Comment:
No semantic change: string cannot be null in this case.  DeMorgan transform from (!a && !b) to !(a || b).